content/en/blog/2022/prow-and-tide-for-kubernetes-contributors/prow-and-tide-for-kubernetes-contributors.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+title: Prow and Tide for Kubernetes Contributors
+date: 2022-05-31
+slug: prow-and-tide-for-kubernetes-contributors
+---
+
+**Authors:** [Chris Short](https://github.com/chris-short)
+
+---
+
+In my work in the Kubernetes world, I look up a label or Prow command often. The systems behind the scenes ([Prow](https://prow.kubernetes.io/) and [Tide](https://pkg.go.dev/k8s.io/test-infra/prow/cmd/tide#section-readme)) are here to help Kubernetes Contributors get stuff done.
+
+Labeling which SIG, WG, subproject, etc. is as important as the issue or PR having someone assigned. To quote [the docs](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/README.md), "Tide is a [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md) component for managing a pool of GitHub PRs that match a given set of criteria. It will automatically retest PRs that meet the criteria ('tide comes in') and automatically merge them when they have up-to-date passing test results ('tide goes out')."
```

Comment:
```suggestion
Labeling which SIG, WG, or subproject is as important as the issue or PR having someone assigned. To quote [the docs](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/README.md), "Tide is a [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md) component for managing a pool of GitHub PRs that match a given set of criteria. It will automatically retest PRs that meet the criteria ('tide comes in') and automatically merge them when they have up-to-date passing test results ('tide goes out')."
```

Reply:
the two sentences in this paragraph don't really go together at all; one is about remembering your SIG label, and the other is what Tide is.  Maybe re-arrange the first part of the article some?

Reply:
Also, a paraphrase of the docs (e.g. "Tide is a collection of Prow actions that merge PRs" or similar) would be more helpful than direct quoting.